core/templates/CRM/Afform/Page/AfformBase.tpl

```diff
@@ -1,9 +1,5 @@
-<h3>This new page is generated by CRM/Afform/Page/AfformBase.php</h3>
-
-{* Example: Display a variable directly *}
-<p>The current time is {$currentTime}</p>
-
-{* Example: Display a translated string -- which happens to include a variable *}
-<p>{ts 1=$currentTime}(In your native language) The current time is %1.{/ts}</p>
-
-<p>Please open "{$afform}"</p>
\ No newline at end of file
+{literal}
+  <div ng-app="crmApp">
+    <form ng-view></form>
```

Comment:
@totten as clever as this is, it just occurred to me that not all afforms may actually be submittable forms. Some may be more like views.

Reply:
@colemanw Is there any drawback to having the `<form>` tag in that scenario?

Also: consider a screen with interactive filtering. It may not modify any DB content, and it may not be submittable in the sense of a traditional business form, but it would still use a composition of form elements in an HTML/technical sense (`<input>`, `<select>`, etc).